cookbooks/letsencrypt/templates/default/request.erb

```diff
@@ -2,6 +2,13 @@
 
 # DO NOT EDIT - This file is being maintained by Chef
 
+if [ "$(id -un)" != "letsencrypt" ] || [ "$(id -gn)" != "letsencrypt" ]; then
+  echo "Error: must be running as user/group 'letsencrypt'"
+  exit 1
+fi
```

Comment:
I can guess why you've added this but it doesn't actually have anything to do with the PR as such...

Reply:
I've removed this for now. It was paranoid check if being run manually. If manually run as root it can screwup permissions for later automatic runs.

Reply:
Actually they do have it because it gets saved in the config on first renewal.